.github/workflows/firefox-ios-update_credential_provider_script.yml

```diff
@@ -41,3 +48,27 @@ jobs:
         title: Refactor [vXXX] auto update credential provider script
         branch: update-cred-provider-script
         token: ${{ secrets.GITHUB_TOKEN }}
+    - name: Send custom JSON data to Slack workflow
+      if: always()
```

Comment:
I think this step will always run even without this line.

Reply:
what I saw is that if for some reason one of the steps above fail and so the github action, the notification step will not run.. but I will dobule check and remove if not needed. Thanks!

Reply:
Aha. I see. Actually [according to this](https://stackoverflow.com/a/58859404) you are correct. If the previous step fails then this won't run. Let's leave it as is then  or do `if: '!cancelled()'` instead so that we don't get pinged when the build was cancelled.